### PR TITLE
Remove zilla load command

### LINKED ISSUE #214: Remove `zilla load` now that we have `zilla metrics` instead
Depends on https://github.com/aklivity/zilla/issues/213.

With extensible `metrics` in place, we no longer need the static counters recorded for `zilla load` command.

Remove `zilla load` command from `incubator` and also any `engine` data structures or APIs used only to support `zilla load` command.

### PR DESCRIPTION
**Pull request by attilakreiner**
## Description

With extensible metrics in place, we no longer need the static counters recorded for zilla load command.

This change removes zilla load command from incubator and also all engine data structures or APIs used only to support zilla load command.

Fixes #214

### CODE REVIEW

File: runtime/binding-tcp/src/test/java/io/aklivity/zilla/runtime/binding/tcp/internal/streams/ClientIT.java [hunk 1]
```diff
@@ -209,20 +207,6 @@ public void shouldEstablishConnectionIPv6() throws Exception
...
-    })
-    public void connnectionFailed() throws Exception
-    {
-        k3po.finish();
-        Thread.sleep(250); // TODO: engine quiesce instead of close
-        EngineStats stats = engine.stats("test", "app0");
-        fullFence();
-        assertEquals(1, stats.initialErrors());
-    }
-
```
Comment (jfallows) [inline runtime/binding-tcp/src/test/java/io/aklivity/zilla/runtime/binding/tcp/internal/streams/ClientIT.java:225]: I think we can keep this test, just without the error counter check, agree?

File: runtime/binding-tcp/src/test/java/io/aklivity/zilla/runtime/binding/tcp/internal/streams/ServerIT.java [hunk 2]
```diff
@@ -378,82 +375,4 @@ public void shouldReceiveDataAfterSendingEnd() throws Exception
...
-        channel3.close();
-        channel5.close();
-        Thread.sleep(500);
-        assertEquals(4, stats.initialOpens());
-        assertEquals(4, stats.initialCloses());
-        assertEquals(4, stats.replyOpens());
-        assertEquals(4, stats.replyCloses());
-
-        k3po.finish();
-    }
```
Comment (jfallows) [inline runtime/binding-tcp/src/test/java/io/aklivity/zilla/runtime/binding/tcp/internal/streams/ServerIT.java:458]: Let's keep this without the stats check.

File: runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/stream/Target.java [hunk 3]
```diff
@@ -62,7 +59,7 @@ public final class Target implements AutoCloseable
     private final Long2ObjectHashMap<LongHashSet> streamSets;
     private final Int2ObjectHashMap<MessageConsumer>[] throttles;
     private final MessageConsumer writeHandler;
-    private final LongFunction<LoadEntry> supplyLoadEntry;
+    //private final LongFunction<LoadEntry> supplyLoadEntry;
```
Comment (jfallows) [inline runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/stream/Target.java:62]: Please remove.

**Review by jfallows — CHANGES_REQUESTED** [review]

File: runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/stream/Target.java (hunk 3 above)
Comment (attilakreiner) [reply runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/stream/Target.java]: done

File: runtime/binding-tcp/src/test/java/io/aklivity/zilla/runtime/binding/tcp/internal/streams/ServerIT.java (hunk 2 above)
Comment (attilakreiner) [reply runtime/binding-tcp/src/test/java/io/aklivity/zilla/runtime/binding/tcp/internal/streams/ServerIT.java:458]: done, pls chk

File: runtime/binding-tcp/src/test/java/io/aklivity/zilla/runtime/binding/tcp/internal/streams/ClientIT.java (hunk 1 above)
Comment (attilakreiner) [reply runtime/binding-tcp/src/test/java/io/aklivity/zilla/runtime/binding/tcp/internal/streams/ClientIT.java:225]: done

**Review by jfallows — APPROVED** [review]

**Pull request merged**